app/views.py

```diff
@@ -99,3 +104,61 @@ def send_email():
 @app.errorhandler(404)
 def page_not_found(error):
     return "oops.. page not found 😝"
+
+
+@app.route("/admin/services")
+def services_management():
+    services = services_model.all()
+    return render_template("admin/services/service_list.html", services=services)
+
+
+@app.route("/admin/services/adding", methods=["GET", "POST"])
```

Comment:
can we change the route to:
`admin/services/add`

Reply:
also instead of having the same route to do both getting and posting data, I prefer to have a separate route to save data, we can name it `save` and keep this one just to present data